src/platform/utilities/environment/index.js

```diff
@@ -55,6 +55,19 @@ if (!isPort80) {
   environment.BASE_URL = LOCALHOST_ENV.BASE_URL;
 }
 
+if (environment.BUILDTYPE === ENVIRONMENTS.LOCALHOST) {
+  // __API__ is defined the same way as __BUILDTYPE__, and is used to indicate the URL of the VA API. The main use
+  // case for this at the moment is for internal review instances to pass configuration during the build.
+
+  try {
```

Comment:
Why do we need the try-catch?

Reply:
I encountered an error during unit tests, `ReferenceError: __API__ is not defined`. I'm thinking we would need to initialize this value in Mocha for it to work, and I didn't want that property to be in too many places since it has such a specific use case. Do you think there's a better way?

Reply:
I think you can do `global.__API__ = null;` to set it.

Reply:
Made that change!